rasa/cli/interactive.py

```diff
@@ -40,11 +66,41 @@ def interactive(args: argparse.Namespace):
 
     args.finetune = False  # Don't support finetuning
 
+    training_files = [
+        get_validated_path(f, "data", DEFAULT_DATA_PATH) for f in args.data
+    ]
+    story_directory, nlu_data_directory = data.get_core_nlu_directories(training_files)
+
+    if not os.listdir(story_directory) or not os.listdir(nlu_data_directory):
+        print_error(
+            "Cannot train initial Rasa model. Please provide NLU data and Core data."
+        )
+        return
+
     zipped_model = train.train(args)
-    model_path = model.unpack_model(zipped_model)
-    args.core, args.nlu = model.get_model_subdirectories(model_path)
-    stories_directory = data.get_core_directory(args.data)
 
-    do_interactive_learning(args, stories_directory)
+    if zipped_model:
```

Comment:
Should we add a debug message in case there is no zipped model? Or can the case even happen that we don't have a zipped model and it didn't crash before? :-D